host_linux.go

```diff
@@ -5,6 +5,8 @@
 package host
 
 import (
-	// Make sure sysfs drivers are registered.
+	// Make sure required drivers are registered.
+	_ "periph.io/x/host/v3/gpioioctl"
+	_ "periph.io/x/host/v3/netlink"
```

Comment:
Let's do one change at a time. This change is already very large. Let's do netlink modification in a separate PR. Thanks.

Reply:
OK, I've reverted those files. I'll re-open the issue I had earlier and make a new PR for it.